forge-gui/res/cardsfolder/upcoming/zeriam_golden_wind.txt

```diff
@@ -0,0 +1,8 @@
+Name:Zeriam, Golden Wind
+ManaCost:3 W
+Types:Legendary Creature Griffin
+PT:3/4
+K:Flying
+T:Mode$ DamageDone | ValidSource$ Griffin.YouCtrl | ValidTarget$ Opponent | CombatDamage$ True | TriggerZones$ Battlefield | Execute$ TrigToken | TriggerDescription$ Whenever a Griffin you control deals combat damage to a player, create a 2/2 white Griffin creature token with flying.
```

Comment:
`ValidTarget$ Player`

always try and reproduce the wording as close as possible for all corner cases
here the combat damage could be redirected back to you and you should still get the trigger